[2021-04-25 13:25:53,247] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-25T12:00:00+00:00 [queued]>
[2021-04-25 13:25:53,283] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-25T12:00:00+00:00 [queued]>
[2021-04-25 13:25:53,284] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 13:25:53,285] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2021-04-25 13:25:53,286] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 13:25:53,298] {taskinstance.py:1063} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc> on 2021-04-25T12:00:00+00:00
[2021-04-25 13:25:53,304] {standard_task_runner.py:52} INFO - Started process 1312 to run task
[2021-04-25 13:25:53,310] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'create_cluster_dataproc', '2021-04-25T12:00:00+00:00', '--job-id', '108', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmprzn2ahzy', '--error-file', '/tmp/tmpjicb5hij']
[2021-04-25 13:25:53,314] {standard_task_runner.py:77} INFO - Job 108: Subtask create_cluster_dataproc
[2021-04-25 13:25:53,372] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-25T12:00:00+00:00 [running]> on host 67976f37379c
[2021-04-25 13:25:53,449] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-25T12:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-25T12:00:00+00:00
[2021-04-25 13:25:53,452] {dataproc.py:603} INFO - Creating cluster: ephemeral-recent-played-spark-cluster-20210425
[2021-04-25 13:30:19,988] {dataproc.py:547} INFO - Cluster created.
[2021-04-25 13:30:20,161] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=create_cluster_dataproc, execution_date=20210425T120000, start_date=20210425T132553, end_date=20210425T133020
[2021-04-25 13:30:20,298] {local_task_job.py:146} INFO - Task exited with return code 1
